modules/plugin/coverage-multidim/netcdf/src/test/java/org/geotools/coverage/io/netcdf/NetCDFUnitParserTest.java

```diff
@@ -159,10 +160,10 @@ public void reset() {
         public void testDU() throws Exception {
             // could not find a way to make this work with the above test, so doing it another way
             Unit<?> unit = NetCDFUnitFormat.parse("DU");
-            assertEquals("µmol·(1/m²)*446.2", unit.toString());
+            assertEquals("(1/m²)*446.2·μmol", unit.toString());
```

Comment:
The research center sponsoring the changes behind this test required exact matches on expected unit expressions. Is there no way to make it look as before? Why did the factors move?

Reply:
This might need a little investigating, The unit library just seems to sort them in a particular way, even though the _netcdf-unit-replacements.properties_ had a replacement that matched the original output the unit library moved it to the back.

Reply:
Here is where I think the documentation would need to be updated.
https://docs.geoserver.org/latest/en/user/extensions/netcdf/netcdf.html#supporting-custom-netcdf-units